include/slang/ast/expressions/AssertionExpr.h

```diff
@@ -91,12 +96,22 @@ class SLANG_EXPORT AssertionExpr {
     /// and false otherwise.
     bool admitsEmpty() const;
 
+    bool isAdmitsOnlyEmpty() const { return admitsOnlyEmpty; }
```

Comment:
Can you rework these three functions into a single one that returns the admits empty / no-match properties via a single flags enum? You can move the `AdmitsEmpty` enum up here and expand it to include a bit for the no-match part as well.  Then you won't need the clunky extra `admitsOnlyEmpty` member below that is only validly set when you call admitsEmpty first, which seems like a footgun.

Reply:
Thanks for suggestion, fixed)!